### PR TITLE
Trader map in new incubator branch with redesign (just bring the map to the new version, not visible yet)

### PR DESCRIPTION
**Pull request by darkblackside**

### CODE REVIEW

**Comment by darkblackside** [comment]
Would be nice of you if someone checks the functionality if it is as before. Currently I've added a map which can be commented in on the trader overview

**Comment by darkblackside** [comment]
Please merge into incubator if everything is working, so we have a almost working map

File: src/app/customer/trader-overview/trader-overview.component2.html [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
+  <!--<div id="map-underlay">
+    <lk-map
+      class="tradersmap"
+      (locationsOnMap)="updateLocations($event)"
+      [traderLocations]="traderLocations"
+      [position]="currentLocation"
+      (markerClicked)="traderpinClicked($event)"
+    ></lk-map>
+  </div>-->
+  <div class="traders-overview-row" id="//map-overlay" *ngIf="hasLocations">
```
Comment (patsimm) [inline src/app/customer/trader-overview/trader-overview.component2.html:17]: I've never seen those two slashes // before an id. Is that some convention I am not aware of?

**Comment by darkblackside** [comment]
Closed in favour to #62

**Pull request closed without merging**